index.html

```diff
@@ -1112,19 +1079,19 @@
               id="q131"
               data-name="Skills"
               class="q131"
-            ><div key="0">Skills</div>
+            ><div>Skills</div>
             </div>
             </div>
             <div class="outer_q132">
             <div
               id="q132"
               data-name="• Codekit • Codepen • GitHub • GitLAb • Terminal"
               class="q132"
-            ><div key="0">• Codekit</div>
-<div key="1">• Codepen</div>
-<div key="2">• GitHub</div>
-<div key="3">• GitLAb</div>
-<div key="4">• Terminal</div>
+            ><div>• Codekit</div>
+<div>• Codepen</div>
+<div>• GitHub</div>
+<div>• GitLAb</div>
+<div>• Terminal</div>
```

Comment:
- [ ] [OPTIONAL] Consider making every list implement this way in `<ul>` tags as pointed out in the review thread.